source/views/building-hours/detail/schedule-row-android.tsx

```diff
@@ -17,24 +17,27 @@ type Props = {
 	now: Moment
 }
 
-export class ScheduleRow extends React.PureComponent<Props> {
-	render() {
-		let {set, isActive, now} = this.props
-		return (
-			<View style={styles.scheduleRow}>
-				<StyledText style={[styles.scheduleDays, isActive && styles.bold]}>
-					{summarizeDays(set.days)}
-				</StyledText>
-
-				<StyledText style={[styles.scheduleHours, isActive && styles.bold]}>
-					{formatBuildingTimes(set, now)}
-				</StyledText>
-			</View>
-		)
-	}
+export const ScheduleRow = (props: Props): JSX.Element => {
+	let {set, isActive, now} = props
+	return (
+		<View style={styles.scheduleRow}>
+			<StyledText style={[styles.scheduleDays, isActive && styles.bold]}>
+				{summarizeDays(set.days)}
+			</StyledText>
+
+			<StyledText style={[styles.scheduleHours, isActive && styles.bold]}>
+				{formatBuildingTimes(set, now)}
+			</StyledText>
+		</View>
+	)
+}
+
+interface StyledTextProps {
+	children: string
```

Comment:
```suggestion
	children: Text['children']
```

maybe? I just feel like there's a better type than "text"…

Reply:
Nice! You made me realize I incorrectly typed `style` as well. Changes pushed.

```diff
interface StyledTextProps {
-	children: string
-	style: StyleProp<ViewStyle>
+	children: Text['props']['children']
+	style: StyleProp<TextStyle>
}